run-benchmark.sh

```diff
@@ -1032,22 +1061,27 @@ runQueries() {
 	      checkBenchResults $SCHEMA $i
 
 	      let i++
-	    done
+	  done
 
-	    # TOODO: behavior: if iterations > 1, provide avg/min/max/stddev
-	  fi
+	  # TODO: behavior: if iterations > 1, provide avg/min/max/stddev
+
+      BENCH_END=$(date -u +'%Y-%m-%d %H:%M:%S')
+      echo "$SPLICEMACHINE_VERSION|$SPLICEMACHINE_IMPL|UNKNOWN_CLUSTER_ID|$BENCH|$SCALE||$ITER|$BENCH_PASS|$BENCH_START|$BENCH_END" > ${LOGBASE}logs/run.csv
+      sort -o ${LOGBASE}logs/test_run.csv ${LOGBASE}logs/test_run.csv
```

Comment:
These files need to go to LOGDIR

Reply:
run.csv, test_run.csv are consumed by Jenkins. It doesn't know about LOGDIR and there may be many LOGDIR's as Jenkins mounts ${LOGBASE}logs.